docs/source/pages/get_started.md

```diff
@@ -0,0 +1,183 @@
+of subject names), with optional session name and data type inputs. If no session or data type name is provided,
+an empty subject directory will be made at the top directory level.
+
+All subject or session names must be prefixed with "sub-" or "ses-" respectively, as according to SWC-BIDS. If these prefixes
+are not input, they will be automatically added. To make all data types (i.e. all datatypes for which use_<data_type> configuration is true),
+ data_type="all" can be set.
+
+Convenience tags (@TO@, @DATE@, @TIME@, @DATETIME@) for creating ranges of subjects, or automatically including
+date-xxxxxx or time / datetime key-value pairs in subject or session names are also available.
+
+An example call:
+
+```
+project.make_sub_dir(
+sub_names=["001", "002"],
+ses_names=["ses-001", "002"],
+data_type=["ephys", "behav", "histology"]
+)
+```
+or equivalently
+
+```
+datashuttle \
+my_project \
+make_sub_dir \
+--sub-names 001 002 \
+--ses-names ses-001 002
+--data_type ephys behav histology
+```
+
+will create the folder structure:
+
+```
+.
+└── my_project/
+    └── rawdata/
+        ├── sub-001/
+        │   ├── ses-001/
+        │   │   ├── ephys
+        │   │   └── behav
+        │   ├── ses-002/
+        │   │   ├── ephys
+        │   │   └── behav
+        │   └── histology
+        └── sub-002/
+            ├── ses-001/
+            │   ├── ephys
+            │   └── behav
+            ├── ses-002/
+            │   ├── ephys
+            │   └── behav
+            └── histology
+```
+
+## Data Transfer
+
+Data transfer can be either from the local project to the remote project ("upload") or from the remote to local project("download"). Data
+transfers are primarily managed using the upload_data() and download_data() functions.
+
+By default, uploading or downloading data will never overwrite files when transferring data. If an
+existing file with the same name is found in the target directory, even if it is older, it will not be overwritten.
+All transfer activity is printed to the console and logged (see "Logging" below), which can be used to
+determine if any files were not transferred for this reason.
+
+For example,
+
+```
+project.download_data(
+sub_names=["all"],
+ses_names=["ses-001", "ses-005"],
+data_type="behav"
+)
+```
+
+or equivalently
+
+```
+datashuttle \
+my_project \
+download_data \
+--sub-names all
+--ses-names ses-001 ses-005
+--data-type behav
+```
+
+will only transfer behavioral data type folders, for sessions 1 and 5 from all subjects.
+
+Along with the @TO@ flag, a wildcard @*@ flag can also be used in subject or session names.
+
+
+## Logging
+
+Detailed logs of all configuration changes, full paths to created directories and data transfers are logged
+to a .datashuttle directory in the local project directory. These logs are named
+with the command (e.g. make_config_file), date and time of creation.
```

Comment:
Not the place to review this? But is there one log file per command? Does this not get unwieldy quickly? Why not just one rotating log file, or one per session/day?